traffic_ops/testing/api/v14/user_test.go

```diff
@@ -95,20 +100,46 @@ func UpdateTestUsers(t *testing.T) {
 func GetTestUsers(t *testing.T) {
 	_, _, err := TOSession.GetUsers()
 	if err != nil {
-		t.Fatalf("cannot GET users: %v\n", err)
+		t.Errorf("cannot GET users: %v\n", err)
 	}
 }
 
 func GetTestUserCurrent(t *testing.T) {
 	user, _, err := TOSession.GetUserCurrent()
 	if err != nil {
-		t.Fatalf("cannot GET current user: %v\n", err)
+		t.Errorf("cannot GET current user: %v\n", err)
 	}
 	if user.UserName == nil {
-		t.Fatalf("current user expected: %v actual: %v\n", SessionUserName, nil)
+		t.Errorf("current user expected: %v actual: %v\n", SessionUserName, nil)
 	}
 	if *user.UserName != SessionUserName {
-		t.Fatalf("current user expected: %v actual: %v\n", SessionUserName, *user.UserName)
+		t.Errorf("current user expected: %v actual: %v\n", SessionUserName, *user.UserName)
+	}
+}
+
+// ForceDeleteTestUsers forcibly deletes the users from the db.
+func ForceDeleteTestUsers(t *testing.T) {
```

Comment:
Do you foresee ever having to use this outside of this test file?

Reply:
no -- I don't..    There shouldn't normally be any way to bypass the normal mechanism of deleting a user of setting role to `disabled`.   This is only required for this test to get things to a clean state.

Reply:
Ok, right on. I was thinking we should make it private so that it's unavailable to the other tests, but then I realized that wouldn't do anything since all the tests are in the same package.